src/Containers/PictureCard.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import styled, { DefaultTheme } from 'styled-components';
+import { StyledIcon } from 'styled-icons/types';
+import { flex, position } from '@Utils/Mixins';
+import { Card as C } from './Card';
+
+interface TagProps {
+    icon: StyledIcon;
+    text: string;
+}
+
+interface PictureCardProps extends MainInterface, ResponsiveInterface {
+    image: string;
+    tags?: TagProps[];
+    theme: DefaultTheme;
+    alt?:  string;
+    height?: string;
+    width?:  string;
+}
+
+export const PictureCard: React.FC<PictureCardProps> = ({
+    children,
+    image,
+    tags,
+    theme,
+    alt,
+    height,
+    width,
+    ...cardProps
+}): React.ReactElement => {
+    return (
+        <Card width={width} height={height} {...cardProps}>
+            <ImageWrapper>
+                <Image src={image} alt={alt} height={height} width={width} />
+                {tags && (
+                    <Tags>
+                        {tags.map(
+                            ({ icon, text }): React.ReactElement => (
+                                <Tag>
+                                    {icon && <Icon as={icon} hasText={text} />}
+                                    {text}
+                                </Tag>
+                            ),
+                        )}
+                    </Tags>
+                )}
+            </ImageWrapper>
+            {children && <Content>{children}</Content>}
+        </Card>
+    );
+};
+
+interface CardProps{
+    width?:string;
+    height?:string;
+}
+
+const Card = styled(C)<CardProps>`
+    width: ${({width}) => width ? width : '300px'};
+    height:${({height}) => height ? height : 'auto'};
+    padding:4px;

```

Comment:
No padding and border-radius should come from theme

Reply:
It was coming theme padding from Card.tsx If I don't change it, the spacing between Card and Picture will come back